lineman/app/models/group_model.coffee

```diff
@@ -17,6 +17,13 @@ angular.module('loomioApp').factory 'GroupModel', (BaseModel, AppConfig) ->
       @hasMany 'subgroups', from: 'groups', with: 'parentId', of: 'id'
       @belongsTo 'parent', from: 'groups'
 
+    proposals: ->
```

Comment:
Hm, the discussion gets included because the previous proposal card includes the discussion's title? I wonder if we could serialize out `discussionTitle` instead of having to include the whole discussion, the way we do with `parentAuthorName` on a comment..